rebar.config

```diff
@@ -25,6 +25,7 @@
 {deps, [
     {cowlib,      ".*", {git, "https://github.com/extend/cowlib.git",           {tag, "1.0.0"}}},
     {leo_commons, ".*", {git, "https://github.com/leo-project/leo_commons.git", {tag, "1.1.2"}}},
+    {leo_logger,            ".*", {git, "https://github.com/leo-project/leo_logger.git",            {tag, "1.1.6"}}},
     {meck,        ".*", {git, "https://github.com/eproxus/meck.git",            {tag, "0.8.2"}}}
```

Comment:
There is a implicit rule that `leo_logger` should be imported at only `leo_(gateway|storage|manager)` to make other libraries less dependent and allow users of libraries to choose their own `erlang_logger` event handler.

Please check the below page to find out how erlang_logger works.
http://www.erlang.org/doc/man/error_logger.html